### PR TITLE
POWR-1809 Pattern Lab Typography Demos

### PR DESCRIPTION
**Pull request by joekarasek**
This PR includes:
- moving some "core" scss assets (loaded first, does not directly print css rules) into the core directory
- removes Bootstraps "_root.scss" which added css vars to <html>. We are not using css vars and they are not supported by IE
- Sans Serif font family defined in theme_variables, bootstrap overrides uses the cloudy variable
- Consolidated Typographic demos in Pattern Lab

Testing:
1. Go to https://powr-1809-portlandor.pantheonsite.io/pattern-lab/?p=viewall-elements-02-typography
2. Review the typography documentation: Is it clear? Is it comprehensive? Is it accurate?
3. Go to https://powr-1809-portlandor.pantheonsite.io
4. Is the site typography the same? Does it match what you saw in Pattern Lab? Is it still loading our font family of choice (Open Sans)?

### CODE REVIEW

File: web/themes/custom/cloudy/src/css/_overrides.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@ Description: Bootstrap 4 with Sass
 /* TYPOGRAPHY */
 
 /* Google Fonts */
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,700,800');
```
Comment (brittanysmart) [inline web/themes/custom/cloudy/src/css/_overrides.scss]: This needs imported into PL.

Comment (joekarasek) [reply web/themes/custom/cloudy/src/css/_overrides.scss]: Good catch, the font was cached on my end

File: web/themes/custom/cloudy/pattern-lab/_patterns/00-elements/02-typography/20-blockquote.twig [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <blockquote>
 	<p>A block quotation (also known as a long quotation or extract) is a quotation in a written document, that is set off from the main text as a paragraph, or block of text, and typically distinguished visually using indentation and a different typeface or smaller size quotation.</p>
-</blockquote>
\ No newline at end of file
+  <footer>Aldous Huxley, <cite>Brave New World</cite>
```
Comment (joshuami) [inline web/themes/custom/cloudy/pattern-lab/_patterns/00-elements/02-typography/20-blockquote.twig]: This is missing a closing tag.

File: web/themes/custom/cloudy/pattern-lab/_patterns/00-elements/02-typography/25-lists.twig [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <ol>
+      <li>This is a nested list item</li>
+      <li>This is another nested list item in an ordered list</li>
+    </ol>
+  </li>
+  <li>This is the last list item</li>
+</ol>
+
+<h3>Definition List</h3>
+<dt>Definition List Title</dt>
```
Comment (joshuami) [inline web/themes/custom/cloudy/pattern-lab/_patterns/00-elements/02-typography/25-lists.twig]: Doesn't a definition title `<dt>` and definition item `<dd>` need to be wrapped in a definition list `<dl>`?

**Pull request merged**